README.rst

```diff
@@ -45,6 +45,8 @@ on the independent variable axis. For example, for powder diffraction data taken
 chemical phases where the measurements were done at different temperatures and the materials were undergoing thermal
 expansion.
 
+I think it is not very clear here what the package does and how it can be used to solve the example problem. You are talking about VT-PXRD experiment of samples with different phases which undergo a termal expansion, so a modification of unit cell parameters, each phase independently with respect to the other in the same sample?
```

Comment:
@Filippo1598 this is a comment, not a fix. Please put comments in the comments section on GitHub and only fixes in the code itself.  What would be needed here is your proposed text.   I'm this case we may want to discuss it a bit so drafting text in the comments section and then editing it over there before pasting it here may be a preferred workflow.